### PR TITLE
Bounty UI updates

### PR DESCRIPTION
**Pull request by mattcasey**

### CODE REVIEW

File: components/bounties/components/BountySubmissionReviewActions.tsx [hunk 1]
```diff
@@ -91,15 +91,16 @@ export default function BountySubmissionReviewActions (
           <>
             <Button
               color='primary'
+              size='small'
               onClick={() => {
                 if (submission.status === 'applied') {
-                  setReviewDecision({ decision: 'approve', submissionId: submission.id, userId: user?.id as string });
```
Comment (mattcasey) [inline components/bounties/components/BountySubmissionReviewActions.tsx:96]: @Devorein I think you had the logic flipped here.. it was asking me to confirm my review when i clicked "Approve" for an application with no submission yet.

File: components/common/CharmEditor/components/Placeholder.tsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ function PlaceHolder ({ show, text = "Type '/' for commands", sx }: { sx?: SxPro
   return show ? (
     <Box sx={{
       top: '-2em',
+      pointerEvents: 'none',
```
Comment (mattcasey) [inline components/common/CharmEditor/components/Placeholder.tsx:11]: this seemed to fix the issue for me that i couldn't select the editor

File: components/bounties/components/BountySubmissionReviewActions.tsx (hunk 1 above)
Comment (Devorein) [reply components/bounties/components/BountySubmissionReviewActions.tsx:96]: Yes I figured that out while doing some QA tests yesterday night and fixed it. But thanks for letting me know

File: components/common/CharmEditor/components/Placeholder.tsx (hunk 2 above)
Comment (Devorein) [reply components/common/CharmEditor/components/Placeholder.tsx:11]: Awesome

**Pull request merged**